redisinsight/ui/src/pages/autodiscover-cloud/column-definitions/components/DatabaseCell/DatabaseCell.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+import { formatLongName, replaceSpaces } from 'uiSrc/utils'
+import { RiTooltip } from 'uiSrc/components'
+import { CellText } from 'uiSrc/components/auto-discover'
+import styles from 'uiSrc/pages/autodiscover-cloud/redis-cloud-databases/styles.module.scss'
+
+import { DatabaseCellProps } from './DatabaseCell.types'
+
+export const DatabaseCell = ({ name, className }: DatabaseCellProps) => {
+  const cellContent = replaceSpaces(name.substring(0, 200))
+
+  return (
+    <div
+      role="presentation"
+      data-testid={`db_name_${name}`}
+      className={className}
+    >
+      <RiTooltip
+        position="bottom"
+        title="Database"
+        className={styles.tooltipColumnName}
```

Comment:
nit: It's a new code, so we can replace the SASS styles with a Styled Component, if it's a simple class.

PS: I see the same pattern in the other files as well, so we can deal with it in a separate PR.